lib/Token/Token.cpp

```diff
@@ -232,16 +244,26 @@ std::string Token::getHash() const {
   hash += this->getName().str();
   hash += this->getFunctionName();
   hash += this->getMemTypeName();
-  hash += this->getFieldIndex();
+//  hash += this->getFieldIndex();   
```

Comment:
If we remove the field index from the hash then we may not be able to differentiate between a and a[1].

Reply:
Actually in LFCPA we don't want to differentiate between a and a[1].